packages/storefront-lwc/src/modules/commerce/storeFront/storeFront.html

```diff
@@ -17,10 +17,10 @@
                     <commerce-product-detail></commerce-product-detail>
                 </lwce-route>
                 <lwce-route default>
-                    <h1 style="color: black;">404 Not found</h1>
+                    <commerce-error-page pagenotfound="true"></commerce-error-page>
```

Comment:
Nit picky, but `pagenotfound="true"` is the same thing as `pagenotfound`. Just the attribute being present implies it to be truthy.

Reply:
Done